stereo/src/stereo_adaptive_cost_so.cpp

```diff
@@ -36,167 +36,188 @@
  *
  */
 
-/** \brief Adaptive Cost 2-pass Scanline Optimization Stereo Matching algorithm implementation
-  * please see related documentation on stereo/stereo_matching.h
-  *
-  * \author Federico Tombari (federico.tombari@unibo.it)
-  * \ingroup stereo
-  */
+/** \brief Adaptive Cost 2-pass Scanline Optimization Stereo Matching algorithm
+ * implementation please see related documentation on stereo/stereo_matching.h
```

Comment:
Looks weird. Also doxygen has related functionality [`@see`](http://www.doxygen.nl/manual/autolink.html)

Reply:
Hm, indeed this docstring has no effect I think, at least I can not find it in the generated documentation. Will wipe it out.